src/main/java/org/folio/reshare/index/storage/Storage.java

```diff
@@ -36,6 +43,10 @@
   public Storage(Vertx vertx, String tenant) {
     this.pool = TenantPgPool.pool(vertx,tenant);
     this.bibRecordTable = pool.getSchema() + ".bib_record";
+    this.matchTypeTable = pool.getSchema() + ".match_type";
```

Comment:
We should align table names with the OpenAPI spec:

* `match_type` -> `MATCH_KEY_CONFIGS`
* `match_point` -> `MATCH_KEY_VALUES`

would be one way. But we can also go the other way and align the OpenAPI spec. @adamdickmeiss @nielserik preferences?